thundra/plugins/metric/metric_plugin.py

```diff
@@ -29,10 +29,10 @@ def __init__(self):
     def before_invocation(self, data):
         self.reporter = data['reporter']
         context = data['context']
-        event = data['event']
         stat_time = time.time() * 1000
         self.stat_data = {
             'transactionId': data['transactionId'],
+            'rootExecutionAuditContextId': data['contextId'],
```

Comment:
Might this be problematic if `contextId` is not exist in `data` when trace plugin is not enabled (and `contextId` is generated by trace plugin as suggested at later review comment)?